version-embed.go

```diff
@@ -6,8 +6,12 @@ package tailscaleroot
 
 import _ "embed"
 
+// VersionFileDoNotUseDirectly is the contents of VERSION.txt. Do not use this
+// directly to figure out the version of the binary, use the version package
+// instead.
+//
 //go:embed VERSION.txt
-var Version string
+var VersionFileDoNotUseDirectly string
```

Comment:
kinda gross name. I don't like prescriptive stuff like this. I'd rather name it what it is. What is it?

Reply:
The bug was that unstable builds, even built correctly with version embedding, would report 1.37.0 as their MajorMinorPatch despite the correct version information existing in the binary. Easy mistake to make because that top level package has a juicy sounding "Version" that must surely be the version, only it's not, it's the literal contents of VERSION.txt, which is only loosely related to the actual build version.

The more literal name would be something like `VersionFileContents`, but then I still have to use the comment to say that the version file contents is almost certainly not what you want to use if you're hoping to get version information. Do you prefer that?

Reply:
Or `VersionTxtBranchOrReleaseVersionFileContents` if you want something long and descriptive rather than prescriptive.  Because it's either a branch (1.35.0) or an explicit stable release version number.

Reply:
How's this?